apps/docs/app/.well-known/did.json/route.ts

```diff
@@ -0,0 +1,27 @@
+import { NextResponse } from "next/server";
+
+export async function GET(request: Request) {
```

Comment:
Since we didn't cover this earlier: what's going on here with duplicating the .well-known configs from GM?

Reply:
It's just to allow subscribing, this will be replaced with its own config later on